support/reftest/reftest

```diff
@@ -38,15 +38,24 @@ class DBHandler:
     ):
         if not from_date:
             from_date = self.default_from_date
-        return self.dynamodb.put_item(
-            TableName=self.table,
-            Item={
+        if not content_type:
+            item = {
```

Comment:
Seems like it'd be better to not duplicate the setting of most of the item fields, since that invites bugs where someone updates one path and forgets the other.

Would it make more sense to write it like this:

```python
    item = {
       "web_uri": # ...,
       # ...rest of fields except content_type
    }
    if content_type:
        item["content_type"] = {"S": content_type}
```

So that there are not two places where web_uri, object_key etc are being set.